pkg/ingester/ingester.go

```diff
@@ -577,7 +578,28 @@ func (i *Ingester) removeShutdownMarkerFile() {
 func (i *Ingester) loop() {
 	defer i.loopDone.Done()
 
-	flushTicker := time.NewTicker(i.cfg.FlushCheckPeriod)
+	// Delay the first flush operation by up to 0.8x the flush time period.
```

Comment:
Appreciate your comment here that explains the behavior. Wondering should we make it clear the same in `cfg.FlushCheckPeriod` config as well.

Rationale being, this avoid surprises for loki operators. Without mentioning this delays(during both initial and every iteration) may set different expectation of the `cfg.FlushCheckPeriod` config IMO.

Reply:
@kavirajk Updated the docs for the `FlushCheckPeriod` setting.